src/utils/validators.js

```diff
@@ -0,0 +1,19 @@
+const validateRequiredEmail = (value) => {
+  let error;
+  if (!value) {
+    error = 'Email address is required';
+  } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(value)) {
+    error = 'Invalid email address';
+  }
+  return error;
+};
+
+const validateRequiredPassword = (value) => {
+  let error;
+  if (!value) {
+    error = 'Password is required';
+  }
+  return error;
+};
+
+export { validateRequiredEmail, validateRequiredPassword };
```

Comment:
This is good practice, and we can keep it as is. But do know that often times going ahead to abstract something preemptively can bite back as it adds complexity.

We have these validators here, but the question is now - where else would we be using these exact validators? Is it worth the developer efficiency? Sometimes the simpliest solution is the best

Reply:
True, just didn't want to clutter the component. I do see it being used in reset password, sign up, etc... in the future or updating a user's email.